webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -164,4 +198,14 @@ export class TasksComponent implements OnInit, OnDestroy {
   listTrackBy(index, task) {
     return task?._id;
   }
+
+  async getCurrentLineageLevel(){
+    return this.userContactService.get().then(user => user?.parent?.name);
+  }
+
+  async getLineagesFromTaskDocs(taskDocs){
+    const ids = _map(taskDocs, 'forId');
+    return await this.lineageModelGeneratorService.reportSubjects(ids);
```

Comment:
```suggestion
    return this.lineageModelGeneratorService.reportSubjects(ids);
```
It returns a promise already, the `await` won't have effect. Remember to remove the `async`